### DISCUSSION TITLE
How to plot cylinderical HSV colorspace?

### DISCUSSION THREAD

**Discussion by devpko** [colour-science/colour#1039] [Q&A]
I want to see to cylinderical HSV colorspace of some image.

Is there any supported function in colour.plotting?

Could you give me any advice?

thanks

**Comment by thomasmansencal** [comment]
Hi @pjaesung2,

Do you have an example of what you are trying to achieve? Is it the polar distribution of some image samples?

Cheers,

Thomas

**Comment by devpko** [comment]
Hello @KelSolaar ,
What I'm trying to do is that calculate color volume of some images and compare them.
![color_volume0](https://user-images.githubusercontent.com/21931228/188851513-aed1343d-856c-4d83-bf81-18297a4fc91f.jpg)
Is there any supported API that calculates convex hull volume of scattered points which is in Lab space?

**Reply by thomasmansencal** [reply]
Hi @pjaesung2,

The colour distribution of an image can be highly concave and the result you would get out of that process would be meaningless.

![image](https://user-images.githubusercontent.com/99779/193436445-aafa3c6f-594a-4dc8-b9fd-bd085f242d68.png)
